datafusion/datasource-csv/src/file_format.rs

```diff
@@ -666,8 +666,10 @@ impl DisplayAs for CsvSink {
                 write!(f, ")")
             }
             DisplayFormatType::TreeRender => {
-                // TODO: collect info
-                write!(f, "")
+                if !self.config.file_groups.is_empty() {
```

Comment:
I debugged a bit more and it seems the issue is that the actual filename target is in `self.config.table_path` rather than self.config_file_groups

However, this way is consisent with the other data sinks

@irenjj  is there any chance you can make a ticket / PR to fix this?